DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -913,6 +915,7 @@ globus_l_gfs_iRODS_start(
                 }
                 iRODS_handle->user = strdup(token);
             }
+            iRODS_handle->first_write_done = false;
```

Comment:
Should we initialize this value to `false` outside of the conditional block? The memory allocated on 841/843 doesn't seem to be set, so it may be a good idea to just zero out the struct, in which case this value would be `false` anyway. Thoughts?

Reply:
Oh, y'know what... if it's no a C-struct, that might not be a good idea. In any case, perhaps this value should be initialized/set outside of the conditional block.

Reply:
I moved the initialization to outside the conditional block although I believe that first strtok() will always return a non-null pointer.

I am hesitant to zero initialize the entire struct because there are mutexes in it and I am not sure if that will cause problems.